integration/tests/unowned_cluster/unowned_cluster_test.go

```diff
@@ -287,7 +287,7 @@ var _ = Describe("(Integration) [non-eksctl cluster & nodegroup support]", func(
 	})
 
 	It("supports deleting clusters", func() {
-		if !params.SkipDelete {
```

Comment:
:100:

Reply:
actually this is right too

@aclevername i suggest that this PR has no merit!

Reply:
this should check for `true`, so it is doing the right thing

Reply:
Shouldn't it just be `SkipDelete || SkipCreate` like if I skip create then I probably want to skip delete? (I'm struggling to imagine when I want to `SkipCreate` in the first place tbh...)

Reply:
oh yeh actually nvm

Reply:
i read it wrong

Reply:
I don't get the use case for the various Skips. But it makes sense to only skip deleting the *cluster* if `SkipDelete` is true